sgkit/tests/test_display.py

```diff
@@ -58,6 +59,60 @@ def test_display_genotypes():
     assert expected_html in disp._repr_html_()
 
 
+def test_display_genotypes__variant_ids():
+    ds = simulate_genotype_call_dataset(n_variant=3, n_sample=3, seed=0)
+    # set some variant IDs
+    ds["variant_id"] = (["variants"], np.array(["V0", "V1", "V2"]))
+    ds["variant_id_mask"] = (["variants"], np.array([False, False, False]))
+    disp = display_genotypes(ds)
+    assert (
+        str(disp)
```

Comment:
I like using [textwrap.dedent](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) in these situations. Can write
```python
required = """\
   samples    S0   S1   S2
   ￼variants               
   ￼V0        0/0  1/0  1/0
   ￼V1        0/1  1/0  0/1
   ￼V2        0/0  1/0  1/1
"""
assert str(disp) == textwrap.dedent(required)
```

Saves the indentation of the file and writing some noqas

Reply:
Great tip! I've changed the tests to use `dedent`. I think the `noqa` is still needed though since there is trailing whitespace on the `variants` line.